chart/testdata/frobnitz/templates/wordpress.jinja

```diff
@@ -0,0 +1,72 @@
+#helm:generate dm_template
```

Comment:
We have a wordpress example [here](https://github.com/kubernetes/deployment-manager/tree/master/examples/wordpress). Can we reuse it instead of building a new one?